src/core/BufferGeometry.js

```diff
@@ -597,6 +597,17 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 		var position = this.attributes.position;
 
+		if ( position && position.isGLBufferAttribute ) {
```

Comment:
How about instructing the user that the have to set `mesh.frustumCulled` to `false` instead?

Reply:
We kind of do that already sometimes when using `SkinnedMesh` or when running gpu particle simulations.

Reply:
>How about instructing the user that the have to set mesh.frustumCulled to false instead?

...or, alternatively, set the bounding sphere manually...

Reply:
Hmm, I didn't know about `mesh.frustumCulled`, this is very useful to me, thanks.

I think here it's better to check if the mesh is not culled or if it has some manual bbox/sphere. Because we don't pull the vertices from GPU to calc the bbox, but user might still need it. The warning is to be shown when both culling enabled and no bbox given. I'll try to implement this soon.

Reply:
I've simply added the warnings in f8f0b41. Now the example is set up to use either custom bounding sphere or no culling. I've checked that it shows the warning if I use neither of those in the example.